extensions/wikia/TemplateDraft/controllers/TemplateDraftController.class.php

```diff
@@ -58,6 +58,36 @@ public function markTemplateAsNotInfobox() {
 		return true;
 	}
 
+	/**
+	 * Get the converted infobox markup of a given template.
+	 *
+	 * @requestParam string template The name of the template to convert.
+	 */
+	public function getInfoboxMarkup() {
+		$this->response->setFormat( WikiaResponse::FORMAT_JSON );
+
+		$templateName = $this->request->getVal( 'template' );
+		$title = Title::newFromText( $templateName, NS_TEMPLATE );
+		if ( !$title instanceof Title
+			|| !$title->exists()
+			|| !$title->inNamespace( NS_TEMPLATE )
```

Comment:
I think we want to use [TemplateDraftHelper::allowedForTitle](https://github.com/Wikia/app/blob/CE-2126-templatedraft-clicktracking/extensions/wikia/TemplateDraft/TemplateDraftHelper.class.php#L90)
once its merged (https://github.com/Wikia/app/pull/7737)